### DISCUSSION TITLE
Home vs Away Arm

### DISCUSSION THREAD

**Discussion by ryanjohn80** [dgreif/ring#1436] [General]
Hi, in the automations for HomeKit there is no way to execute an automation based off Home vs Away arming of the alarm.  The only actions are "Triggers" "Disarms" "Arms"- is there away to get an Arm Home & Arm Away that anyone knows of.  Thanks vm

**Comment by tsightler** [comment]
This Reddit thread discusses the same thing and provides options:

https://www.reddit.com/r/homebridge/comments/pcgng3/drive_automation_based_on_away_or_stay_arm_on/

**Comment by ryanjohn80** [comment]
Amazing thank you very much. Worked like a charm. On Jun 21, 2024, at 5:08 PM, tsightler ***@***.***> wrote:﻿
This Reddit thread discusses the same thing and provides options:
https://www.reddit.com/r/homebridge/comments/pcgng3/drive_automation_based_on_away_or_stay_arm_on/
